src/miral/minimal_window_manager.cpp

```diff
@@ -355,8 +355,7 @@ bool miral::MinimalWindowManager::Impl::handle_pointer_event(MirPointerEvent con
     {
         if (auto const window = tools.window_at(new_cursor))
         {
-            if (tools.active_window() != tools.select_active_window(window))
-                consumes_event = true;
+            tools.select_active_window(window);
```

Comment:
I'm not sure (beyond "history") why the logic was this way, but we probably need to change the corresponding case in `handle_touch_event()` to match. I need to try that out...

Reply:
`handle_touch_event()` updated